src/components/Dashboard/Summary/Summary.js

```diff
@@ -41,6 +41,16 @@ function Summary(props) {
               day="numeric"
             />
           ),
+          nearly:
+            percentage > 0 ? (
```

Comment:
uh this was an earlier change that i made that wanted to send a PR about and I ended up sending it here 😅 it just removes the `nearly` when the percentage is 0. Otherwise for vestings that had never done a release the text said `[...] of said fund, nearly 0% of what is currently vested was released.`. Now when the percentage is exactly 0 it reads `[...] of said funds 0% of what is currently vested was released`.